ffmpeg/ffmpeg.go

```diff
@@ -120,15 +121,26 @@ func accelDeviceType(accel Acceleration) (C.enum_AVHWDeviceType, error) {
 
 func Transcode2(input *TranscodeOptionsIn, ps []TranscodeOptions) error {
 
+	_, ret := Transcode3(input, ps)
+	return ret
+}
+
+type TranscodeReceipt struct {
```

Comment:
Can this be named `TranscodeResults` rather than `TranscodeReceipt` ?

Reply:
done in 23abea821757539ba27052094946ceb5d5d01f0e